### DISCUSSION TITLE
unequal length time series Error

### DISCUSSION THREAD

**Discussion by ruipinghe** [kernc/backtesting.py#428] [Q&A]
Could I use an unequal length time series as input data? My data is looks like this:
DatetimeIndex(['2011-06-05 14:20:32', '2011-06-05 14:21:21',
               '2011-06-05 14:23:47', '2011-06-05 14:26:29',
               '2011-06-05 14:27:40', '2011-06-05 14:28:53',
               '2011-06-05 14:30:34', '2011-06-05 14:31:54',

the spaces between two bars are not equal, I got a error : Invalid value for `superimpose`: Upsampling not supported.
But I don't want it to be resampled, is there any way to solve this problem?

**Comment by kernc** [comment]
Without resampling to a fixed period, not as long as https://github.com/kernc/backtesting.py/issues/233 is not fixed.

**Reply by ruipinghe** [reply]
So without resampling, I can't use it?
